src/main/java/oncall/domain/date/Date.java

```diff
@@ -0,0 +1,29 @@
+package oncall.domain.date;
+
+public class Date {
+    private final Month month;
+    private final int day;
+    public final DayOfWeek dayOfWeek;
+
+    public Date(Month month, int day, DayOfWeek DayOfWeek) {
```

Comment:
파라미터는 소문자로 시작하면 좋겠네요!
```suggestion
    public Date(Month month, int day, DayOfWeek dayOfWeek) {
```

Reply:
허걱,, 이것도 알아차리지 못했네요,, 감사합니다!!